### PR TITLE
VPN-4721 - Update qtglean docs page with daemon info

### PR DESCRIPTION
**Pull request by brizental**

### CODE REVIEW

File: docs/qtglean.md [hunk 1]
```diff
@@ -262,6 +263,25 @@ Since that is linked to the Mozilla VPN project the Glean platform specific code
...
+while backgrounded. This caveat is a big issue for telemetry.
+
+A normal usage pattern on mobile is to open the application, turn on the VPN, background it and not
+interact with it for a long time. All of this time of usage is not capture by the main application
+Glean instance. Read more about this issue on [Overview and implications of scheduled task behavior on mobile devices](sarahs-doc)
+and ["Mozilla VPN Telemetry Refactor](beas-doc).
+
+To deal with that, the mobile daemons (Android Daemon and iOS Network Extension, to be more precise)
+have a completely separate instance of Glean that permits telemetry collection while the application
+is backgrounded. This choice of architecture, means there is not need to deal with the complexities of passing
```
Comment (Gela) [inline docs/qtglean.md:279]:
```suggestion
is backgrounded. This choice of architecture means there is no need to deal with the complexities of passing
```

File: docs/qtglean.md [hunk 2]
```diff
@@ -262,6 +263,25 @@ Since that is linked to the Mozilla VPN project the Glean platform specific code
...
+telemetry between processes -- the daemons are each run in a separate process from the main application.
```
Comment (Gela) [inline docs/qtglean.md:280]:
```suggestion
telemetry between processes -- the daemons each run in a separate process from the main application.
```

File: docs/qtglean.md [hunk 3]
```diff
@@ -271,3 +291,5 @@ any call to a Glean API from any language will record to the same Glean instance
 [t-c-g-ffi]: https://searchfox.org/mozilla-central/source/toolkit/components/glean/api/src/ffi
 [glean-bool-set]: https://mozilla.github.io/glean/book/reference/metrics/boolean.html
 [uniffi]: https://mozilla.github.io/uniffi-rs/
+[sarahs-doc]: https://docs.google.com/document/d/1A2O_eACk0P3jDy0g2rRN5UTBOUj-iN-re6tVsRpxtDA/edit?usp=sharing
```
Comment (Gela) [inline docs/qtglean.md:294]: This document is marked as confidential, do we still want to expose the link?

Comment (brizental) [reply docs/qtglean.md:294]: The link will only work for NDA'd people anyways... It provides useful context for all of this, but I can remove it. Do you think that is problematic?

Comment (Gela) [reply docs/qtglean.md:294]: I personally don't mind this, if there are no other concerns I think we can leave it as is

**Review by Gela — APPROVED** [review]

**Pull request merged**